portfolio/src/main/webapp/style.css

```diff
@@ -32,4 +32,7 @@
   color: blanchedalmond;
 }
 
-
+#homelinks img {
+    width: 50px;
```

Comment:
inconsistent spacing for the css with the other formatting.

also for css, its convention to use two spaces instead of four.